### ISSUE TITLE
freetype fails for x64-windows-static because pkgconfig cannot be found

### ISSUE THREAD

**Issue by m-kuhn** [microsoft/vcpkg#25790]
**Host Environment**
- OS: Windows

**To Reproduce**
`./vcpkg install freetype:x64-windows-static`

**Failure logs**

```
cmd /c "cd .. && "C:/Users/Public/QField/vcpkg/base/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/bin/cmake.exe" "C:/Users/Public/QField/vcpkg/base/buildtrees/freetype/src/6482de1748-c0b4903603.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/Users/Public/QField/vcpkg/base/packages/freetype_x64-windows" "-DFT_DISABLE_HARFBUZZ=ON" "-DFT_REQUIRE_ZLIB=ON" "-DFT_REQUIRE_BZIP2=ON" "-DFT_REQUIRE_PNG=ON" "-DFT_REQUIRE_BROTLI=ON" "-DFT_DISABLE_ZLIB=OFF" "-DFT_DISABLE_BZIP2=OFF" "-DFT_DISABLE_PNG=OFF" "-DFT_DISABLE_BROTLI=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/Public/QField/vcpkg/base/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Users/Public/QField/vcpkg/base/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Users/Public/QField/vcpkg/base" "-D_VCPKG_INSTALLED_DIR=C:/Users/Public/QField/build-static/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF""
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE) 
-- The C compiler identification is MSVC 19.29.30133.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Found ZLIB: C:/Users/Public/QField/build-static/vcpkg_installed/x64-windows/lib/zlib.lib (found version "1.2.12") 
-- Found PNG: C:/Users/Public/QField/build-static/vcpkg_installed/x64-windows/lib/libpng16.lib (found version "1.6.37") 
-- Found BZip2: C:/Users/Public/QField/build-static/vcpkg_installed/x64-windows/lib/bz2.lib (found version "1.0.8") 
-- Looking for BZ2_bzCompressInit
-- Looking for BZ2_bzCompressInit - found
CMake Error at C:/Users/Public/QField/vcpkg/base/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find BrotliDec (missing: BROTLIDEC_LIBRARIES)
Call Stack (most recent call first):
  C:/Users/Public/QField/vcpkg/base/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  builds/cmake/FindBrotliDec.cmake:51 (find_package_handle_standard_args)
  C:/Users/Public/QField/vcpkg/base/scripts/buildsystems/vcpkg.cmake:841 (_find_package)
  CMakeLists.txt:288 (find_package)
```

**Additional context**
freetype should depend on vcpkg-pkgconfig-get-modules
```
    {
      "name": "vcpkg-pkgconfig-get-modules",
      "host": true
    }
```

see https://github.com/freetype/freetype/blob/master/builds/cmake/FindBrotliDec.cmake#L22

**Comment by dg0yt** [comment]
FTR it is not `vcpkg-pkgconfig-get-modules`. That port only provide portfile functions.

**Comment by m-kuhn** [comment]
You are right, that didn't fix it, sorry for the quick shot

**Comment by m-kuhn** [comment]
Might only appear in manifest mode or certain dependency resolution orders. Fix supplied in #25957 , follows the approach in ports/gts/portfile.cmake and many more

**Comment by LilyWangLL** [comment]
Since this issue can't be reproduced stably, so I am closing this issue for now. If someone could reproduce this issue, please provide the repro steps and reopen this issue.
